static/app/components/arithmeticInput/parser.tsx

```diff
@@ -1,5 +1,10 @@
+import {t} from 'app/locale';
+
 import grammar from './grammar.pegjs';
 
+const MAX_OPERATORS = 10;
```

Comment:
Can we add a comment that this should stay in sync with the constant in the backend?